.pre-commit-config.yaml

```diff
@@ -179,8 +179,10 @@ metastore_browser/templates/.*\\.html$|.*\\.jinja2"
     rev: 5.0.2
     hooks:
       - id: pydocstyle
+        name: Run pydocstyle
         args:
-          - --select=D300,D301,D302,D101,D201,D206,D207,D208,D209,D211,D414
+          - --convention=pep257
```

Comment:
This is the default for pydocstyle. Even if I remove this line, it won't make any difference